pkgs/applications/misc/blender/default.nix

```diff
@@ -80,6 +81,16 @@ stdenv.mkDerivation rec {
         --prefix PYTHONPATH : ${pythonPackages.numpy}/${python.sitePackages}
     '';
 
+  # Set RUNPATH so that libcuda and libnvrtc in /run/opengl-driver(-32)/lib can be
+  # found. See the explanation in libglvnd.
+  postFixup = optionalString cudaSupport ''
+    for program in $out/bin/blender $out/bin/.blender-wrapped; do
+      isELF "$program" || continue
+      origRpath=$(patchelf --print-rpath "$program")
+      patchelf --set-rpath "$origRpath:${libGL_driver.driverLink}/lib" "$program"
```

Comment:
Maybe we want to put `driverLink` path first instead? This way if a package depends on Mesa or nvidia drivers directly it would still load system-wide drivers, not the ones it was linked with -- that's the behavior we arguably expect. It's especially important with nvidia because its driver libraries require exactly the same kernel module version and provide stable ABI.

Reply:
There is only one libGL that can be linked to, that is the one from `libglvnd`. Neither the `mesa_drivers` nor `nvidia-x11` provide a libGL that can be linked to, only `libGL_vendor` that is dlopen-ed from `libglvnd`. I do not see any case where this RPATH adjustment would need to override some default library. Adding the RPATH at the end should reduce the performance impact to normal library loading.